tests/acceptance/features/cliPasswordChange/occResetPasswordLastPasswords.feature

```diff
@@ -12,14 +12,12 @@ Feature: enforce the number of last passwords that must not be used when resetti
       | username | password |
       | user1    | Number1  |
 
-  @skipOnOcV10.2
```

Comment:
Actually we do not need to skip these on core `10.2` - the output comes the same to `stderr` on both `10.2` and `10.3`